app/src/main/report/components/ReportsList/Filter/ReportsListFilterDate.tsx

```diff
@@ -1,53 +0,0 @@
-import * as React from "react";
-
-import {ReportsFilterFields} from "../../../actionTypes/ReportsActionsTypes";
-import {DatePicker} from "../../../../shared/components/DatePicker/DatePicker";
-import {CalendarIcon} from "../Calendar";
-
-interface ReportsListFilterProps {
-    filterData: ReportsFilterFields;
-    timeFromSelected: (date: Date) => void;
-    timeUntilSelected: (date: Date) => void;
-}
-
-const fromMonth = new Date("2017-03-01T00:00:00");
-const toMonth = new Date;
```

Comment:
Also pulled these values out of this component, instead passing them in as props.